### DISCUSSION TITLE
Using multiple slices and a global store, is it possible to persist only specific slices?

### DISCUSSION THREAD

**Discussion by chrber04** [pmndrs/zustand#1375] [Q&A] [answered]
As the title suggests, I am curious if it's possible to use the slices pattern detailed here:

https://github.com/pmndrs/zustand/pull/1344/files

While only applying the persist middleware to specific slices? I know one solution is to use multiple stores, but as far as I understand it, it's preferrable to have only one store.

Thanks!

**Comment by dai-shi** [comment, marked as answer]
Please try `partialize` option.
